hexes/hexes.py

```diff
@@ -51,6 +54,26 @@ def __repr__(self):
         ).format(s=self)
 
     @property
+    def text(self):
+        return self._text
+
+    @text.setter
+    def text(self, val):
+        dirty = val != self._text
```

Comment:
... Which is then a candidate for a descriptor of its own, at which point: exeunt boilerplate